ignite/pkg/cosmosgen/templates/root/package.json.tpl

```diff
@@ -10,23 +10,24 @@
       "url": "http://www.apache.org/licenses/LICENSE-2.0"
     }
   ],
-  "main": "index.ts",
+  "main": "lib/index.js",
   "publishConfig": {
     "access": "public"
   },
-  "dependencies": {
-    "@cosmjs/launchpad": "0.27.0",
-    "@cosmjs/proto-signing": "0.27.0",
-    "@cosmjs/stargate": "0.27.0",
+  "scripts": {
+    "build": "NODE_OPTIONS='--max-old-space-size=16384' tsc"
+  },
+  "dependencies": {    
+    "@cosmjs/proto-signing": "0.31.1",
+    "@cosmjs/stargate": "0.31.1",
     "@keplr-wallet/types": "^0.11.3", 
     "axios": "0.21.4",
     "buffer": "^6.0.3",
     "events": "^3.3.0"
   },
   "peerDependencies": {
-    "@cosmjs/launchpad": "0.27.0",
-    "@cosmjs/proto-signing": "0.27.0",
-    "@cosmjs/stargate": "0.27.0"
+    "@cosmjs/proto-signing": "0.31.1",
```

Comment:
Curious, does it fixes https://github.com/ignite/cli/issues/3686 ?

Reply:
yep...

Reply:
yep...

Reply:
I was trying to bump it here but everything was breaking: https://github.com/ignite/web 😅

Reply:
I need to document the integrations tbh